build/images.txt

```diff
@@ -2,6 +2,7 @@
 ../official/19.0.0.x/kernel/java8/ibmsfj         kernel-java8-ibmsfj
 ../official/19.0.0.x/kernel/java8/openj9         kernel-java8-openj9
 ../official/19.0.0.x/kernel/java11/openj9        kernel-java11
+../official/19.0.0.x/kernel/java12/openj9        kernel-java12
```

Comment:
I drafted this comment last week (before we had version numbers in our docker images). Re-thinking this, having `java12` in the tag won't be any extra tags than `latestJava`, since we have all tags duplicated for each release anyway.